db/migrate/20160606040848_create_payments.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePayments < ActiveRecord::Migration
+  def change
+    create_table :payments do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :last4
+      t.decimal :amount, precision: 12, scale: 3
+      t.string :authorization_code
+      t.integer :status, default: 0
+      t.integer :user_id, null: false
+      t.integer :conference_id, null: false
+      t.datetime :created_at
+      t.datetime :updated_at
```

Comment:
the `t.timestamps` call on line 15 adds the `created_at` and `updated_at` columns. We don't need to add them again.